Code/Mantid/Framework/PythonInterface/plugins/algorithms/DNSDetEffCorrVana.py

```diff
@@ -1,9 +1,14 @@
-# pylint: disable=no-init
+import sys
+import os
 import mantid.simpleapi as api
 from mantid.api import PythonAlgorithm, AlgorithmFactory, MatrixWorkspaceProperty
 from mantid.kernel import Direction
 import numpy as np
 
+sys.path.insert(0, os.path.dirname(__file__))
```

Comment:
I don't think this is required anymore. This would have been required but was fixed recently after I had issues using inheritance in Python algorithms.